src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  position: relative;
+  font-family: Avenir, Helvetica, sans-serif;
+  margin-top: 0;
+}
+
+.bar-form {
+  position: absolute;
+  width: 100%;
+}
+
+.big-field {
+  position: relative;
+  height: 70px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.small-field {
```

Comment:
no need to repeat all styles and use `big-field` and `small-field`

checklist
<img width="943" alt="image" src="https://user-images.githubusercontent.com/102411130/229055870-457c03c8-dc29-472f-a194-3d29e0b660bf.png">